packages/universal-crossplane/universal-crossplane.patch

```diff
@@ -0,0 +1,22 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/universal-crossplane/charts-original/Chart.yaml packages/universal-crossplane/charts/Chart.yaml
+--- packages/universal-crossplane/charts-original/Chart.yaml
++++ packages/universal-crossplane/charts/Chart.yaml
+@@ -1,8 +1,11 @@
+ apiVersion: v1
+-appVersion: 1.2.1-up.3
++appVersion: 1.2.1003
+ description: Upbound Universal Crossplane allows you to connect your Crossplane-enabled cluster to Upbound Cloud.
+ home: https://upbound.io
+ icon: https://crossplane.io/images/favicon_192x192.png
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: universal-crossplane
```

Comment:
Could you please add this additional annotation for the name that will be displayed in the UI?
`catalog.cattle.io/display-name: Upbound Universal Crossplane`
I used Upbound Universal Crossplane as an example, but you can change it to any other name you think is best.

Reply:
Done, thanks!